docker/build_and_push_ep.Dockerfile

```diff
@@ -88,4 +88,4 @@ ENV LANGFLOW_HOST=0.0.0.0
 ENV LANGFLOW_PORT=7860
 
 USER 1000
-ENTRYPOINT ["python", "-m", "langflow", "run", "--host", "0.0.0.0", "--backend-only"]
+CMD ["python", "-m", "langflow", "run", "--host", "0.0.0.0", "--backend-only"]
```

Comment:
@zzzming Adding comment here to remind us we're making this change here, in case we see issues late when deploying a new image to cloud